pkgs/development/python-modules/uwsgi-chunked/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "uwsgi-chunked";
+  version = "0.1.8";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "btimby";
+    repo = "uwsgi-chunked";
+    tag = version;
+    hash = "sha256-5TNCnQhnT1gAblgs+AAW62HoNDPM54hpxgCnYl07j3I=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "uwsgi_chunked" ];
+
```

Comment:
Tests?

Reply:
Testing this would require setting up a testing instance via docker